### ISSUE TITLE
[DOCUMENT]: GIBCT SCO - Dear SCO letter text SHOULD be linked or adjusted

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8761]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

VA 508 office has identified the [Dear School Certifying Official](https://www.benefits.va.gov/GIBILL/docs/fgib/107_TandF_SCO_letter.pdf) PDF has some text that should either be linked or updated to not appear as links visually. Screen shot attached below.

## Related Issue
* #3918
* #8758 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* [ ] Blue underlined text open the intended URL **or**
* [ ] Text is changed to not resemble a link visually

## Screenshots or Trace Logs
![Screen Shot 2020-05-05 at 11 49 00 AM](https://user-images.githubusercontent.com/934879/81092843-d5cef380-8ec6-11ea-956d-30f7b1bc38a2.png)

**Comment by TamGIBill** [comment]
Letter has been removed.
